tests/integration/README.md

```diff
@@ -544,6 +544,9 @@ The test framework supports the following command-line flags:
   -istio.test.kube.deploy
         Deploy Istio into the target Kubernetes environment. (default true)
 
+  -istio.test.kube.deployEastWestGW
```

Comment:
When would we disable this at the environment level?

Reply:
In case it is not needed, either to isolate issues or reduce unnecessary components.